lib/source_file.cpp

```diff
@@ -69,7 +71,35 @@ void InputSources::addComment(SourceInfo srcInfo, bool singleLine, cstring body)
     if (!singleLine)
         // Drop the "*/"
         body = body.exceptLast(2);
-    comments.push_back(new Comment(srcInfo, singleLine, body));
+    auto srcPos = SourcePosition(srcInfo.getStart());
+
+    comments.push_back(new Comment(srcPos, singleLine, body));
+}
+
+std::string SourceInfo::getComments() const {
+    if ((sources == nullptr)) {
+        return "";
+    }
+
+    std::ostringstream oss;
+    uint targetLine = start.getLineNumber();
```

Comment:
This returns `unsigned`, not `uint`. There is no `uint` standard type.
```suggestion
    unsigned targetLine = start.getLineNumber();
```